src/page-parole/PageParole.js

```diff
@@ -73,12 +73,11 @@ export default function PageParole() {
       tortor vitae iaculis egestas. Donec dictum, nunc nec tincidunt cursus,
       ipsum dui gravida.`,
       showDimensionControl: true,
-      showMonthControl: true,
       VizComponent: VizSupervisionRevocation,
       vizData: {
-        supervisionRevocationByMonth: apiData.supervision_revocations_by_month_by_type_by_demographics.filter(
-          recordIsParole
-        ),
+        revocationsByDemographics: apiData.supervision_revocations_by_period_by_type_by_demographics
+          .filter(recordIsParole)
+          .filter(recordIsMetricPeriodMonths(36)),
```

Comment:
We don't strictly need to filter by the `metric_period_months` attribute because the file I created only has 36-month entries, but this adds in a little safety in case we add more period types into the same file at a later date.